demo-app-vue-3/package.json

```diff
@@ -5,10 +5,11 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit"
+    "test:unit": "vue-cli-service test:unit --watch"
```

Comment:
Do we need `--watch` here? I think not having the `watch` option makes more sense as the default.

Reply:
I will remove it. And do another push.